app/pages/Auth/AuthPage.presenter/AuthPage.presenter.tsx

```diff
@@ -50,27 +55,33 @@ const useStyles = makeStyles((theme: Theme) => ({
 
 const AuthPage: FC = () => {
   const classes = useStyles();
-  const { isAuthenticated, isInitialized } = useFullService();
+  const { isAuthenticated, selectedAccount } = useFullService();
   const [selectedView, setView] = useState(1);
-  const [isUnlocked, setUnlocked] = useState(false);
   const { t } = useTranslation('AuthPage');
+  const [walletDbExists, setWalletDbExists] = useState(localStore.getWalletDbExists());
+  const [fullServiceIsRunning, setFullServiceIsRunning] = useState(false);
+  const [loading, setLoading] = useState(true);
+  const [accountIds, setAccountIds] = useState([]);
 
-  useEffect(() => {}, [isUnlocked]);
-
-  const getWallet = async () => {
+  const checkIfFullServiceIsRunning = async () => {
     try {
-      await getWalletStatus();
-    } catch (e) {
-      // nothing!
+      const status = await getWalletStatus();
+      setAccountIds(status.accountIds);
+      setFullServiceIsRunning(true);
+    } finally {
+      setLoading(false);
     }
   };
-  getWallet();
 
-  if (!isInitialized) {
+  useEffect(() => {
+    checkIfFullServiceIsRunning();
+  }, []);
```

Comment:
Can be written:

`useEffect(checkIfFullServiceIsRunning, []);`

Reply:
prettier formatted it this way and complains otherwise